tests/inspectdb/tests.py

```diff
@@ -121,31 +121,31 @@ def test_number_field_types(self):
 
         assertFieldType('int_field', "models.IntegerField()")
 
-        if connection.features.can_introspect_positive_integer_field:
+        if connection.features.introspected_field_types['PositiveIntegerField'] == 'PositiveIntegerField':
             assertFieldType('pos_int_field', "models.PositiveIntegerField()")
         else:
             assertFieldType('pos_int_field', "models.IntegerField()")
 
-        if connection.features.can_introspect_positive_integer_field:
-            if connection.features.can_introspect_big_integer_field:
+        if connection.features.introspected_field_types['PositiveIntegerField'] == 'PositiveIntegerField':
+            if connection.features.introspected_field_types['BigIntegerField'] == 'BigIntegerField':
                 assertFieldType('pos_big_int_field', 'models.PositiveBigIntegerField()')
```

Comment:
I believe you can simplify all this stuff to lines like:
```
assertFieldType('pos_big_int_field', 'models.%s() % connection.features.introspected_field_types['PositiveBigIntegerField'])
```
I don't think the if statements are needed anymore (similar elsewhere in this file).